sites/fast-component-explorer/package.json

```diff
@@ -65,7 +65,7 @@
     "react-dnd": "^9.0.0",
     "react-dom": "^16.6.3",
     "react-router-dom": "^5.1.2",
-    "react-syntax-highlighter": "^10.3.0",
+    "react-syntax-highlighter": "^13.3.1",
```

Comment:
I did a little look through the codebase, we don't actually use this anymore since the code preview is now just monaco editor which has. its own syntax highlighting, so you could just remove this dependency.